introduction.md

```diff
@@ -6,3 +6,4 @@ deadly skills:
 * Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?